deploy/docker-compose.yml

```diff
@@ -90,5 +99,13 @@ services:
     command:
       - bash
       - -c
-      # Run the SecOps demo after giving the service some time to initialize
-      - "sleep 5 && cd demo/project_demo00-SecOps && python3 run.py --dbsp_url http://pipeline-manager:8080 --actions prepare create compile $${SECOPS_DEMO_ARGS}"
+      # Run the demos after giving the service some time to initialize
+      # Start the two demos in parallel.  Compilation will still happen serially, but
+      # this way the user will see the two programs and pipelines straight away.
+      - "sleep 1\ 
+
+        \ (cd demo/project_demo00-SecOps && python3 run.py --dbsp_url http://pipeline-manager:8080 --actions prepare create compile $${SECOPS_DEMO_ARGS})&\ 
```

Comment:
What exactly is the exit code behavior when running both these commands in the background like this? Also, what gets logged to stdout/stderr?

Reply:
They both write to the console, so we'll see logs from both.  The exit status is 0, which .. may not be what we want.  So, what do we want this container to do?  Exit when one of the demos fails or keep running the other one?

Reply:
We run integration tests in the Earthfile where we rely on the container's exit code. See for example: https://github.com/feldera/feldera/blob/cec50ad3f81478c3ff2b57fdc4962b9887bde600/Earthfile#L535.

We'd need to make sure the exit codes are reliable in this case. So if even one of the demos fails, it should reflect as such in the container's exit code.

Reply:
Fair enough